nodejs/builder/Dockerfile-debian

```diff
@@ -0,0 +1,10 @@
+ARG BUILDER_IMAGE=fission/builder
+FROM ${BUILDER_IMAGE}
+FROM node:8
```

Comment:
Node v8 is already out of date. Last time we had added some v12 images but I think the current stable is v15 so we better use it I guess (https://nodejs.org/en/about/releases/) Also same would apply to previous comments about sticking to a specicic version

Reply:
Yeah, I kept it the same as default ones. We should probably drop support for very old versions.

Reply:
https://nodejs.org/en/about/releases/, version 10, 12, 14, 15 should be added.